src/CodeGen_WebAssembly.cpp

```diff
@@ -1,15 +1,19 @@
-#include "CodeGen_Posix.h"
+#include <functional>
+#include <sstream>
 
+#include "CodeGen_Posix.h"
+#include "ConciseCasts.h"
+#include "IRMatch.h"
+#include "IROperator.h"
 #include "LLVM_Headers.h"
-#include <sstream>
 
 namespace Halide {
 namespace Internal {
 
-using std::string;
```

Comment:
Did you mean to change this here? Most of the other backends do this (and don't quality string with std::).

Reply:
There was a mixture of usage so I normalize to std:: since that is the pattern in most non-backend code. Will normalize the other way if that's the unspoken standard here.